aioxmpp/xso/model.py

```diff
@@ -2807,3 +2820,50 @@ def to_xso(self):
         instances of their XSO classes) into XSOs.
         """
         return self.xso_class()
+
+
+# XXX: This was move here from aioxmpp/xso/__init__.py
+# and the tests still reside in test___init__.py
```

Comment:
Move the tests to ``test_model.py`` and add a test that ``aioxmpp.xso.AbstractTextChild is aioxmpp.xso.model.AbstractTextChild`` to ``test___init__.py``.